alpine/Dockerfile

```
@@ -60,11 +64,13 @@ RUN apk update --no-cache && \
     rm -rf /var/cache/apk/*
 
 # Limit JDK crypto policy by default to comply with local laws which may prohibit use of unlimited strength cryptography
-RUN JAVA_HOME=$(ls -d /usr/lib/jvm/*jdk*) && \
+RUN JAVA_HOME=$(find /usr/lib/jvm/ -maxdepth 1 -name "*jdk*" -type d) && \
```

```suggestion
RUN JAVA_HOME=$(find /usr/lib/jvm -maxdepth 1 -name "*jdk*" -type d) && \
```

The same command is also used to set the `JAVA_HOME` env var in the entrypoint scripts so it would be best to also update those.

---

Can you also remove the redundant slash?